legacy/google-cloud/setup.py

```diff
@@ -76,9 +76,17 @@
 
 setup(
     name='google-cloud',
-    version='0.32.1.dev1',
+    version='0.32.2',
```

Comment:
ISTM that jumping into the fully-deprecated state should be a non-patch version bump, i.e., `0.33.0`.

Reply:
This version doesn't change any behavior so I thought a patch release made sense.

The following release, which will be in mid June, will result in this package being a no-op install that has none of the libraries anymore.

While this doesn't make any code change, I can also see your view that we should rev a minor version for this. 

With this explanation do you still believe it makes more sense to rev to 0.33 now and 0.34 in June? The plan was to do 0.32.2 now and rev to 0.33.0 in June.  I am going to proactively fix this to be 0.33.0.